wiki/People/Beatmap_Nominators/id.md

```diff
@@ -0,0 +1,172 @@
+[false]: /wiki/shared/false.png "No"
+[true]: /wiki/shared/true.png "Yes"
+
+Beatmap Nominator
+==================
+
+*Lihat juga: [Peraturan Beatmap Nominator](/wiki/Beatmap_Nominator_Rules) dan [Informasi Umum untuk Beatmap Nominator](/wiki/Beatmap_Nominator_General_Information).*
+
+Beatmap Nominator, dikenal juga dengan *BN*, adalah kelompok yang bertanggung jawab atas [penominasian beatmap](/wiki/Submissions/Beatmap_Ranking_Procedure). Sejak 2014-08-22, Beatmap Nominator telah dipisahkan dari bagian staff, perubahan ini meliputi pencabutan hak moderasi di chat in-game, nama merah di in-game, dan di dalam forum. Pada 2015-02-01, nama kelompok berubah dari *Beatmap Appreciation Team* menjadi *Beatmap Nominators*. Saat ini, mereka memiliki warna ungu pada nama mereka di forum.
+
+Peran dan Tanggung Jawab
+--------------------------
+
+Anggota Beatmap Nominator bertanggung jawab terutama untuk [modding](/wiki/Modding) [beatmaps berstatus pending](/wiki/Beatmaps) ke dalam ranah status ranking. Beatmap Nominator adalah orang-orang yang dapat dimintai saran tentang mapping atau modding.
```

Comment:
>[modding](/wiki/Modding) [beatmaps berstatus pending](/wiki/Beatmaps) 

What about `melakukan [modding](/wiki/Modding) terhadap [beatmaps berstatus pending](/wiki/Beatmaps)`? This might be slightly convoluted, but applying English grammar rules to a primarily Indonesian sentence tends to end up making it awkward to read. Might be worth it to try.